deny.toml

```diff
@@ -0,0 +1,10 @@
+[licenses]
```

Comment:
The ["taplo" CI check for YAML formatting](https://github.com/rustls/rustls/actions/runs/14237900888/job/39900863210?pr=2412) is upset (but doesn't say much about why... :thinking: ):

> ERROR taplo:format_files: the file is not properly formatted path="/home/runner/work/rustls/rustls/deny.toml"

Reply:
It wanted me to indent by two (instead of four spaces) -- fixed.